lib/virt_feature_test_base.pm

```diff
@@ -49,7 +50,10 @@ sub run_test {
 sub prepare_run_test {
     my $self = shift;
 
-    reconnect_when_ssh_console_broken unless defined(script_run("rm -f /root/{commands_history,commands_failure}", die_on_timeout => 0));
+    unless (defined(script_run("rm -f /root/{commands_history,commands_failure}", die_on_timeout => 0))) {
+        reconnect_when_ssh_console_broken unless defined(script_run("rm -f /root/{commands_history,commands_failure}", die_on_timeout => 0));
```

Comment:
try another time deliberately or forgot to remove it?? `unless defined(script_run("rm -f /root/{commands_history,commands_failure}", die_on_timeout => 0))`

Reply:
Good catch! I thought I removed that part :(. Will update.

Reply:
done.